### ISSUE TITLE
s3 path without template doesn't copy tiles

### ISSUE THREAD

**Issue by mapsam** [mapbox/mapbox-tile-copy#85]
If I run the following:

```
mapbox-tile-copy file.geojson s3://waka/flocka/
```

I get an appropriate output...

```
8 tiles @ 8/s, 3% complete [0s]
9 tiles @ 9/s, 3% complete [0s]
10 tiles @ 10/s, 4% complete [0s]
11 tiles @ 11/s, 4% complete [0s]
18 tiles @ 18/s, 7% complete [0s]
59 tiles @ 59/s, 27% complete [0s]
172 tiles @ 172/s, 100% complete [0s]
172 tiles @ 172/s, 100% complete [0s]
```

...but no files/folders exist within the s3 bucket/prefix. Some appropriate solutions:

1. Throwing an error requiring a template `{z}/{x}/{y}` is required OR
1. default to `{z}/{x}/{y}` and make this clear in the README

cc @rclark @GretaCB @who8mycakes

**Comment by mapsam** [comment]
Similarly, if you don't provide `{z}/{x}/{y}.extension` you get files without extensions like

`0/0/0` compared to `0/0/0.mvt`

**Comment by rclark** [comment]
I think it should fail if your url template doesn't include `{z}` `{x}` and `{y}`. Lack of extensions is a feature though, not a bug 😉

**Comment by mapsam** [comment]
Resolved in #87
